data/conf/nginx/includes/site-defaults.conf

```diff
@@ -13,14 +13,18 @@
   ssl_session_timeout 1d;
   ssl_session_tickets off;
 
-  add_header Strict-Transport-Security "max-age=15768000;";
+  add_header Strict-Transport-Security "max-age=15768000; preload";
```

Comment:
According to https://hstspreload.org, `preload` has no effect when `includeSubDomains` is not specified. And we definitely don't want that option as it has an effect outside of Mailcow.

Reply:
I agree that it should be removed, a quote also from hstspreload.org:
> If you maintain a project that provides HTTPS configuration advice or provides an option to enable HSTS, **do not include the preload directive by default.**